core/modules/proxy/mysqlProxy.lua

```diff
@@ -3,61 +3,13 @@
 -- for executing queries. While it has some responsibilities now, it
 -- should eventually act as a thin wrapper to delegating all
 -- functionality to a Qserv daemon.
--- For questions, contact Jacek Becla or Daniel L. Wang
-
-require ("xmlrpc.http")
-
--- todos:
---  * support DESCRIBE
---
---  * communication with user, returning results etc
---
---  * improve error checking:
---    - enforce single bounding box
---    - check if there is "objectId=" or "objectId IN" in the section
---      of query which we are skipping
---    - block "SHOW" except "SHOW DATABASES" and "SHOW TABLES"
---
---  * supress errors "FUNCTION proxyTest.areaSpec_box does not exist"
---
---  * test what happens when I change db (use x; later use y)
-
--- API between lua and qserv:
---  * invoke(cleanQueryString, hintString)
---  * cleanQueryString is the query without special hints: objectId
---    related hints are passed through, but bounding boxes are not
---  * hintString: "box", "1,2,11,12", "box", "5,55,6,66",
---    "objectId", "3", "objectId", "5,6,7,8" and so on
 
--------------------------------------------------------------------------------
---                             debug tool                                    --
--------------------------------------------------------------------------------
-local DEBUG = os.getenv('DEBUG') or 0
```

Comment:
Don't you want to keep a native LUA logger for debugging? If yes,  env variable can now be removed from etc/init.d/mysql-proxy and LUA_DEBUG_LEVEL can also be removed.